src/test/scala/com/typesafe/conductr/sbt/SbtTypesafeConductRSpec.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2014 Typesafe, Inc. All rights reserved.
+ */
+
+package com.typesafe.conductr.sbt
+
+import akka.actor.ActorSystem
+import org.scalatest.{ BeforeAndAfterAll, Matchers, PrivateMethodTester, WordSpec }
+
+class SbtTypesafeConductRSpec extends WordSpec with Matchers with BeforeAndAfterAll with PrivateMethodTester {
```

Comment:
Quite hacky, I'd rather `private[conductr]`, but followed style of tests around here